Source/Versions.swift

```diff
@@ -14,6 +14,19 @@ public enum Semantic {
   case Major, Minor, Patch, Same, Unknown
 }
 
+public class App {
```

Comment:
Went with class because of Objective-C compatibility but I guess it's time to drop that.